src/options.adb

```diff
@@ -289,17 +295,20 @@ package body Options is
       P ("  -Wspecs            warns if a all/others spec does not apply");
       P ("  -Wunused           warns if a subprogram is never used");
       P ("  -Werror            turns warnings into errors");
+      P ("  -Wpsl-uncovered    Prints warning after simulation for each");
+      P ("                     uncovered PSL cover point.");
+      P ("                     Only valid with '-fpsl'");
```

Comment:
What's with inline PSL? I think, the option is valid for all kinds of PSL, isn't it?